src/us38.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[6]:
+
+
+from Gedcom import Gedcom
+
+
+# In[7]:
+
+
+from datetime import date, datetime, timedelta
+
+
+# In[58]:
+
+
+def upcomingDate (dt):
+    
+    if dt == '':
+        return False
+    
+    today = datetime.today()
+    upcomingDays = today + timedelta(days = 30)
+    
+    dt = datetime.strptime(dt + " " + date.today().strftime('%Y'), '%d %b %Y')
```

Comment:
Just pushed up a change to Gedcom that take care of parsing datetimes, so don't need to do this anymore